pysidt/sidt.py

```diff
@@ -100,6 +101,12 @@ def __init__(
         elif root_group:
             self.root = Node(root_group, name="Root", depth=0)
             self.nodes = {"Root": self.root}
+            if initial_root_splits:
+                for i,grp in enumerate(initial_root_splits):
```

Comment:
```suggestion
                for i, grp in enumerate(initial_root_splits):
```

Reply:
Can you run the black formatter on this file? It should take care of all of these formatting problems